jfrog-oauth/README.md

```diff
@@ -106,6 +105,25 @@ jf pip install requests
 pip install requests
 ```
 
+### Configure code-server with JFrog extension
+
+The [JFrog extension](https://open-vsx.org/extension/JFrog/jfrog-vscode-extension) for VS Code allows you to interact with Artifactory from within the IDE.
+
+```hcl
+module "jfrog" {
+  source = "https://registry.coder.com/modules/jfrog-oauth"
+  agent_id = coder_agent.example.id
+  jfrog_url = "https://jfrog.example.com"
+  username_field = "username" # If you are using GitHub to login to both Coder and Artifactory, use username_field = "username"
+  configure_code_server = true # Add JFrog extension configuration for code-server
```

Comment:
```suggestion
  configure_code_server = true # Add JFrog extension configuration for code-server
```

thoughts on install_extension = true?

Reply:
On second thought, I think `configure_code_server` is better since it is specific to code-server, not other IDEs such as VS Code